lang/rust/Makefile

```diff
@@ -90,7 +90,6 @@ define Host/Compile
 	TARGET_CFLAGS="$(TARGET_CFLAGS)" \
 	$(PYTHON) $(HOST_BUILD_DIR)/x.py \
 		--build-dir $(HOST_BUILD_DIR)/build \
-		--config $(HOST_BUILD_DIR)/config.toml \
```

Comment:
Your commit looks like ordinary single bump, but this change is not described anywhere. Is it intentional? If yes, please describe this change in further details.

Reply:
Upstream decided to error out if `config.toml` doesn't exists.

Reply:
Let me do further tests though.